dio/lib/src/utils.dart

```diff
@@ -133,3 +143,14 @@ Map<String, V> caseInsensitiveKeyMap<V>([Map<String, V>? value]) {
   if (value != null && value.isNotEmpty) map.addAll(value);
   return map;
 }
+
+void debugLog(String message, StackTrace stackTrace) {
```

Comment:
Can we make this a static set-able property on the Dio class? That way this could be configured. On Flutter a user could use `debugPrint` or a custom logging solution for example.

Reply:
I need to rethink about it as a public API.

Reply:
Started a new discussion https://github.com/cfug/dio/discussions/1982. May I add a TODO to the method first so we can continue rolling this?

Reply:
Sounds good 👍